template.yaml

```diff
@@ -621,3 +621,59 @@ Resources:
       ManagedPolicyArns:
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSBackupServiceRolePolicyForBackup"
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/AWSBackupServiceRolePolicyForS3Backup"
+
+  LambdaVersionCleanupFunctionAccessLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub '/aws/lambda/${LambdaVersionCleanupFunction}'
+      RetentionInDays: 90
+
+  # Lambda function to delete old versions
```

Comment:
Its better to add comments as "Description:" That way we can also view them in AWS

Reply:
old versions of what?

Reply:
resolved